### DISCUSSION TITLE
OpenSUSE - community package opensuse

### DISCUSSION THREAD

**Discussion by DimSystem** [evilsocket/opensnitch#1158] [Q&A]
Iam having some trouble installing the rpm on tumbleweed (for some reason, even after installing the .asc multiple times, it stills does not match for yast). 

So I looked up packages on zypper (https://build.opensuse.org/search?search_text=opensnitch), are any of these official? Anyone had any experience with it?

Thank you!!

**Comment by henrixd7** [comment]
I don't think they are official. I ended up installing from community repository because I just wanted to try this.
```
zypper addrepo https://download.opensuse.org/repositories/home:jaimeMF/openSUSE_Tumbleweed/home:jaimeMF.repo
zypper refresh
zypper install opensnitch
```
I did need to manually enable and start the systemd service:
`sudo systemctl enable opensnitch; sudo systemctl start opensnitch`

Seems to work fine. I don't know if those packages should be trusted though.
